crates/bitwarden-exporters/src/cxf/ssh.rs

```diff
@@ -45,6 +49,41 @@ pub(super) fn to_ssh(
     Ok((ssh, fields))
 }
 
+impl TryFrom<SshKey> for Vec<Credential> {
+    type Error = SshKeyExportError;
+
+    fn try_from(ssh_key: SshKey) -> Result<Self, Self::Error> {
```

Comment:
It make more sense to have try from with reference:
```suggestion
impl TryFrom<&SshKey> for Vec<Credential> {
    type Error = SshKeyExportError;

    fn try_from(ssh_key: &SshKey) -> Result<Self, Self::Error> {
```

Reply:
The other From uses owned types so I'm keeping the existing behaviour for consistency.